packages/hub/services/data-integrity-checks/cron-tasks.ts

```diff
@@ -0,0 +1,102 @@
+import { inject } from '@cardstack/di';
+import { Client } from 'pg';
+import { IntegrityCheckResult } from './utils';
+
+import config from 'config';
+
+export type TaskIdentifier =
+  | 'check-reward-roots'
+  | 'execute-scheduled-payments'
+  | 'remove-old-sent-notifications'
+  | 'print-queued-jobs';
+
+interface TaskConfig {
+  cronMinutes: number;
+}
+
+interface CronTask {
+  'execute-scheduled-payments': TaskConfig;
+  'check-reward-roots': TaskConfig;
+  'print-queued-jobs': TaskConfig;
+  'remove-old-sent-notifications': TaskConfig;
+}
+
+// graphile worker configuration (copied from worker-client)
+const TASK: CronTask = {
+  'execute-scheduled-payments': { cronMinutes: 5 },
+  'check-reward-roots': { cronMinutes: 10 },
+  'print-queued-jobs': { cronMinutes: 5 },
+  'remove-old-sent-notifications': { cronMinutes: 600 },
+};
+
+let DEFAULT_MULTIPLIER = 3;
```

Comment:
In the PR description it says the multiplier is 2. Which one is more suitable?

Reply:
Lets use a multiplier of 3